### ISSUE TITLE
Enhancement: Add explicit mapping for POCO field's type

### ISSUE THREAD

**Issue by not-authorized** [ErikEJ/EFCorePowerTools#815]
### Description
It is possible to store serialized objects to (O)RDBMS (e.g. to nvarchar field in MSSQL, to jsonb field in PostgreSQL). Therefore it'd be helpful to specify the field type in POCO class explicitly and by doing this way to override generic types like nvarchar or jsonb.

For instance, there is a table;
```pgsql
CREATE TABLE account
(
    id                     int generated always as identity,
    first_name             varchar(50)  not null,
    last_name              varchar(50)  not null,
    profile                jsonb,
    PRIMARY KEY (id)
)
```
and appropriate POCO:
```csharp
public partial class Account
{
    public int Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Profile { get; set; } // <-- public Profile Profile { get; set; }
}
```
and instead of `string` in the Profile field, it'd be nice to have `Profile` as a data type.

### Vision
The feature might be implemented by adding another screen. The screen shall display the list of fields for certain table and respective data types. By entering a type name the default data type shall be substituted with the override we entered.

### Further technical details
EF Core Power Tools version: 2.5
Database engine: Postgres
Visual Studio version: 16.8.5

**Comment by ErikEJ** [comment]
Use a partial class to add the Profile column?

**Comment by not-authorized** [comment]
But I already have the appropriate field generated. Need some way to replace it or override it.

**Comment by ErikEJ** [comment]
Exclude it??

**Comment by not-authorized** [comment]
Within IDE? I'm afraid it isn't a good idea because we are a fairly big team and everyone should keep in mind to exclude all the fields across the model (42 tables). The fact that this operation repeats often makes the tool helpful less.
Generated files should not be touched at all.

**Comment by ErikEJ** [comment]
The exclusion is persisted in a config file in source control...

**Comment by not-authorized** [comment]
Thanks for the hint! This feature definitely has to be documented. 👌🏻

[Direct link to the Exclusion usage](https://github.com/ErikEJ/EFCorePowerTools/blob/a5c1cede6a151ddb777d9647146a8aa54ef9522b/src/GUI/efreveng/TestFiles/edmx.config.json#L33)

**Comment by ErikEJ** [comment]
It is already documented: https://github.com/ErikEJ/EFCorePowerTools/wiki/Reverse-Engineering#choose-database-objects - or do you mean the fact that it is saved?

So you are unblocked now?

**Comment by not-authorized** [comment]
My fault. Didn't see it before.

I'm unblocked now. Thanks a lot!

**Comment by ErikEJ** [comment]
I will make it clearer that selections are saved!

**Comment by ErikEJ** [comment]
I have added this sentence to the docs:

Notice that any changes you make in this dialog will be persisted in .config.json files in your project.
